Introduction.md

```diff
@@ -304,4 +304,11 @@ filter :: (a -> Bool) -> [a] -> [a]
 
 ## Hoogle
 
+- Search the libraries by type
 - https://www.haskell.org/hoogle/
+
+## QuickCheck
```

Comment:
Yeah, definitely not what I had in mind. Anything they can't use directly on the night.

There's so many goodies but I'm worried about distractions given the time. :(